css-values-4/Overview.bs

```diff
@@ -2588,7 +2588,7 @@ Trigonometric Functions: ''sin()'', ''cos()'', ''tan()'', ''asin()'', ''acos()''
 	all represent the same value,
 	approximately ''.707''.)
 	They all represent a <<number>>;
-	''sin()'' and ''cos()'' will always return a number between 0 and 1,
+	''sin()'' and ''cos()'' will always return a number between -1 and 1,
 	while ''tan()'' can return any number between +∞ and −∞.
```

Comment:
Off-topic, but while we are at this it seems a bit inconsistent to say "between −1 and 1" (increasing order) and then "between +∞ and −∞" (decreasing order).

```suggestion
	while ''tan()'' can return any number between −∞ and +∞.
```